microsoft/iot-cardboard-js
# OAT: Add toggle for default version number

### Summary of changes 🔍 
Adding a toggle to let users select the default version of new models. 
![image](https://user-images.githubusercontent.com/57726991/218594448-3bdac185-a175-450a-9889-dd942c657319.png)
![image](https://user-images.githubusercontent.com/57726991/218590425-aaee32df-c9a1-406c-a952-4674a72748ae.png)


### Testing 🧪
- use an existing project before the change, add a model, should have version 2 context
- change the toggle to V3, see new models get V3 context
- change toggle to V2, see new models get V2 context

### Checklist ✔️
